### ISSUE TITLE
Blender not exporting .obj

### ISSUE THREAD

**Issue by CADMonkeyCDN** [X-Plane/XPlane2Blender#627]
Hello;

I'm completely new to using xplane2blender and I hope I have the right forum here. I'm having trouble getting Blender 3D to export .obj files. I've downloaded/installed/enabled xplane2blender (various versions) in Blender (various versions), but whenever I try to export to .obj no file is created. I can still save/save as, but I can't import. When I click "Export X-Plane Object" in the top right of my screen with a filename entered Blender simply returns to the 3d viewport.
I'm probably missing something obvious, but I'm using Youtube tutorials and the steps I take always seem to work in the videos.
My most recent attempt was in Blender 2.8 using XPlane2Blender v4.0.0-beta.1, however, I did try previous xplane2blender releases and earlier versions of Blender. 
Please help! Thank you in advance.

**Comment by tngreene** [comment]
Hi, have you had success yet?

**Comment by CADMonkeyCDN** [comment]
I did eventually find a tutorial that explains in detail using a recent version of Blender (2.83.2) on Youtube (https://www.youtube.com/watch?v=x_S9NosDYIM) in conjunction with a newer xplane2blender. There seemed to be some version conflicts between the add-on and Blender in my early attempts to export. On top of that the tutorials I viewed outlined the steps for Blender 2.78/.79, which differ from 2.8 forward, so I had some problems.
I found that exporting an object worked when I followed the instructions described in the video. Very helpful to a beginner such as myself!
